cmd/ocsp-updater/main.go

```diff
@@ -354,7 +367,12 @@ func (updater *OCSPUpdater) findRevokedCertificatesToUpdate(batchSize int) ([]co
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT * FROM certificateStatus
+		`SELECT
```

Comment:
I *think* exploding out this query is to make sure things work when the model doesn't match the current DB schema yet. Is that right?